README.md

```diff
@@ -92,41 +90,41 @@ The supported `boot/iso/` sub-directories (in alphabetical order) are :
 
 [//]: # (distro-list-end)
 
-Any unpopulated directory will have the matching boot menu entry automatically
-disabled, so to skip any distribution, just don't copy any files into it.
+Unpopulated directories turns its matching boot menu entry off,
+so skip any distribution by not copying any files into it.
 
-Download the right ISO image(s) to the matching directory. If you require
-boot parameter tweaks, edit the appropriate `boot/grub2/inc-*.cfg` file.
+Download the right ISO image(s) to the matching directory.
+If you require boot parameter tweaks, edit the appropriate
+`boot/grub2/inc-*.cfg` file.
 
-Items order in the menu
-------------
+Item order in the menu
+----------------------
 
-Menu items for a distro are ordered by modification time of the iso files
-starting from the most recent ones. If some iso files have the same mtime, their
-menu items are ordered alphabetically.
+Distros as menu items are ordered by the modification time of their ISO files,
+starting with the most recent. If some ISO files have the same mtime,
+their menu items are ordered A–Z.
 
-Here is a generic idea how to keep it nicely ordered when you have multiple
+Here is a generic idea of how to keep it nicely ordered with multiple
 releases of some distro:
 
-- touch your **release** iso files with the release date
-- touch your **point release** iso files with the original release date plus a
+- Touch your **release** ISO files with the release date
+- Touch your **point release** ISO files with the original release date, plus a
   day per point. This is a way to ensure point releases never pop above the next
   release like Debian 10.13.0 (released 10 Sep 2022) would still be below Debian
-  11.0.0 (released 14 August 2021)
-- in case there are multiple flavours of some iso but the version is the same,
-  touch all of them with the same date for the whole group to be ordered
-  alphabetically
-
-Sample ordered menu:
+  11.0.0 (released 14 Aug 2021)
+- In case there are multiple flavours of some ISO, but the version is the same
+  —touch all of them with the same date to order the whole group A–Z
```

Comment:
Why that weird special hyphen-ish character? :sweat_smile:

Reply:
an em dash. Removed.